src/services/ledger/send.ts

```diff
@@ -114,6 +135,24 @@ export const send = {
       return new UnknownLedgerError(err)
     }
   },
+
+  revertOnChainPayment: async ({
+    journalId,
+    description = "Protocol error",
+  }: RevertOnChainPaymentArgs): Promise<void | LedgerServiceError> => {
```

Comment:
Is there anything we can return here except `void` (maybe `true` or something similar). I know at one point we were trying to avoid implicit returns of void.

Reply:
I used `revertLightningPayment` as template... should I change that method too?

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/1871/commits/e23a8038437318b7475ab5a2ca00e0d5d2320ed2